src/tcp.c

```diff
@@ -548,23 +548,8 @@ static relpRetVal
 relpTcpTLSSetPrio(relpTcp_t *pThis)
 {
 	int r;
-	char pristringBuf[4096];
-	char *pristring;
 	ENTER_RELPFUNC;
-	/* Compute priority string (in simple cases where the user does not care...) */
```

Comment:
I'm not sure if that's related to the travis failure, but I see that the `if(pThis->pristring == NULL) {` condition is now ignored and the `pThis->pristring` not taken into account. Shouldn't it be something like:
```
if(pThis->pristring != NULL) {
r = gnutls_priority_set_direct(pThis->session, pThis->pristring, NULL);
} else {
r = gnutls_set_default_priority(pThis->session);
}
```

Reply:
Thanks for suggestion, applied!